docs/operation/account_administration.rst

```diff
@@ -8,18 +8,12 @@ changing the default username and password for some MetalK8s services.
 
 .. _ops-grafana-admin:
 
-Administering Grafana
-*********************
+Changing Grafana Username and Password
```

Comment:
Either "Changing the Grafana Username and Password" or "Changing Grafana Usernames and Passwords"